### PR TITLE
chore(dev-deps): upgrade `react-scripts`

### PR DESCRIPTION
**Pull request by eventualbuddha**
These changes were almost entirely about upgrading eslint.

### CODE REVIEW

File: arlo-client/src/components/AuditFlow/index.tsx [hunk 1]
```diff
@@ -37,20 +37,19 @@ const AuditFlow: React.FC<IProps> = ({
   const [audit, setAudit] = useState<IAudit | undefined>(undefined) // here entereth the error monster
```
Comment (MorganLove) [inline arlo-client/src/components/AuditFlow/index.tsx]: This comment can be removed now that particular error monster hath been slain.

File: arlo-client/src/components/AuditForms/CalculateRiskMeasurement.test.tsx [hunk 2]
```diff
@@ -46,10 +48,10 @@ beforeEach(() => {
...
   } as unknown) as jsPDF
   jspdfMock.mockImplementation(() => jspdfInstance)
-  setIsLoadingMock.mockClear()
-  updateAuditMock.mockClear()
-  getStatusMock.mockClear()
-  toastSpy.mockClear()
+  sharedSetIsLoadingMock.mockClear()
+  sharedUpdateAuditMock.mockClear()
+  sharedGetStatusMock.mockClear()
+  sharedToastSpy.mockClear()
```
Comment (MorganLove) [inline arlo-client/src/components/AuditForms/CalculateRiskMeasurement.test.tsx]: Why the name change?

File: arlo-client/src/components/AuditForms/CalculateRiskMeasurement.tsx [hunk 3]
```diff
@@ -199,7 +198,7 @@ const CalculateRiskMeasurement: React.FC<IProps> = ({
           40
         )
         placeholders.text(`Ballot Number: ${ballot.position}`, 20, 100)
-        pageCount++
+        pageCount += 1
```
Comment (MorganLove) [inline arlo-client/src/components/AuditForms/CalculateRiskMeasurement.tsx]: Does the new eslint version now allow `++`? Or is this just a stylistic preference?

File: arlo-client/src/components/AuditForms/CalculateRiskMeasurement.test.tsx (hunk 2 above)
Comment (eventualbuddha) [reply arlo-client/src/components/AuditForms/CalculateRiskMeasurement.test.tsx]: The newer eslint version includes `no-shadow`, which requires that inner scopes not shadow a variable with the same name as one in an outer scope.

File: arlo-client/src/components/AuditForms/CalculateRiskMeasurement.tsx (hunk 3 above)
Comment (eventualbuddha) [reply arlo-client/src/components/AuditForms/CalculateRiskMeasurement.tsx]: The newer ESLint includes [`no-plusplus`](https://eslint.org/docs/rules/no-plusplus) by default.

File: arlo-client/src/components/AuditFlow/index.tsx (hunk 1 above)
Comment (eventualbuddha) [reply arlo-client/src/components/AuditFlow/index.tsx]: Done.

**Pull request merged**